aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -43,14 +45,15 @@
  */
 public class S3FileIO implements FileIO {
   private static final Logger LOG = LoggerFactory.getLogger(S3FileIO.class);
-  private static final String DEFAULT_METRICS_IMPL = "org.apache.iceberg.hadoop.HadoopMetricsContext";
 
   private SerializableSupplier<S3Client> s3;
   private AwsProperties awsProperties;
   private transient S3Client client;
   private MetricsContext metrics = MetricsContext.nullMetrics();
   private final AtomicBoolean isResourceClosed = new AtomicBoolean(false);
 
+  private static final String S3 = "s3";
```

Comment:
variable name should be more specific for the purpose of usage, like `IO_METRICS_SCHEME_DEFAULT`